### PR TITLE
Add links to storyboards in cpm data tabs

### PR DESCRIPTION
**Pull request by geek-yang**
- Use one tab to host links to storyboards
- Fetch storyboards links from one yaml file
- Display links to different stories based on the region of selection

### CODE REVIEW

File: pages/cpm.vue [hunk 1]
```diff
@@ -17,8 +17,22 @@
...
           </button>
         </div>
-        <img src="cpm_analysis/cpm_prec.png" alt="cpm"> <!-- change the path when the figs are ready -->
-        <p> You select the following: {{ domain }}</p>
+        <div v-if="tab.body == 'Overview'" class="p-3">
+          <p> You select the following: {{ domain }}</p>
+          <img src="cpm_analysis/cpm_prec.png" alt="cpm"> <!-- change the path when the figs are ready -->
+        </div>
+        <div v-if="tab.body == 'Storyboards'" class="p-3">
+          <p>Read more about the stroies in this region:</p>
```
Comment (Peter9192) [inline pages/cpm.vue:25]:
```suggestion
          <p>Read more about the stories in this region:</p>
```

File: content/storyboards.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+stories:
+  NW:
+  - title: TBA
+    link: https://eucp-project.github.io/storyboards
+
```
Comment (Peter9192) [inline content/storyboards.yaml:5]: Perhaps at some point, we will be able to fetch these stories directly from the storyboards page, e.g.

```js
stories = await fetch("https://eucp-project.github.io/storyboards/_content/stories?region='alpine'&category='cpm'")
```

see https://content.nuxtjs.org/advanced#api-endpoint

**Review by Peter9192 — APPROVED** [review]
Nice! Perhaps it would be good to display all stories if no region is selected (that's what it starts at for me). Of course the alternative would be to set a default region. Not sure which option is better considering all other tabs.

File: pages/cpm.vue (hunk 1 above)
Comment (geek-yang) [reply pages/cpm.vue:25]: Well spot! Corrected!

File: content/storyboards.yaml (hunk 2 above)
Comment (geek-yang) [reply content/storyboards.yaml:5]: @Peter9192 Thanks for the tip! We will keep it for further developments later.

**Comment by geek-yang** [comment]
> Nice! Perhaps it would be good to display all stories if no region is selected (that's what it starts at for me). Of course the alternative would be to set a default region. Not sure which option is better considering all other tabs.

Sounds great! I've added a default option to load all the stories (note that since the stories in some regions are still "TBA", they are currently served as placeholders rather than actual stories, but easy for further updates.).

**Pull request merged**